public/wallet-service-worker.mjs

```
@@ -6347,6 +6347,10 @@ class vf {
           f = w[w.length - 1];
         }
       } catch (i) {
+        if (Kh(i)) {
+          console.debug("Timeout error ignored");
```

ESLint

[error] Strings must use singlequote.

(@typescript-eslint/quotes)